docs/reference/signals.rst

```diff
@@ -26,8 +26,8 @@ example showing how you might notify your team when something is published:
 .. code-block:: python
 
     from wagtail.core.signals import page_published
-    import urllib
-    import urllib2
+    import requests
```

Comment:
Could you add a comment that indicates that requests is not a part of the standard libraries? See the geocoder example here https://docs.wagtail.io/en/stable/advanced_topics/customisation/page_editing_interface.html#customising-generated-forms

Reply:
`requests` is required by Wagtail, so it might not be important to make the distinction here:

https://github.com/wagtail/wagtail/blob/master/setup.py#L36

Reply:
`requests` is required by Wagtail, so the distinction may not be important here:

https://github.com/wagtail/wagtail/blob/master/setup.py#L36

Reply:
@tomdyson is right, requests is always present. Let's skip this requirement.